gr-azure-software-radio/lib/difi_source_cpp_impl.h

```diff
@@ -91,12 +91,20 @@ class difi_source_cpp_impl : public difi_source_cpp<T>
       memcpy(&imag, start + 1, 1);
       return M(re, imag);
     }
-    std::string d_host_name;
+
+    void create_udp_socket();
+    void create_tcp_socket();
+    void reset_tcp_connection();
+
+    std::string d_ip_addr;
     uint32_t d_port;
     int d_stream_number;
-    uint32_t d_buffer_len;
+    uint32_t d_packet_buffer_len;
     int d_socket;
-    std::vector<int8_t> d_buffer;
+    int d_client_socket;
+    uint8_t d_socket_type;
+    std::vector<int8_t> d_readBuffer;
```

Comment:
stick with snake case

Reply:
this variable is not needed I will remove it